src/modules/tools/zprobe/CartGridStrategy.cpp

```diff
@@ -277,12 +319,13 @@ bool CartGridStrategy::handleGcode(Gcode *gcode)
             // first wait for an empty queue i.e. no moves left
             THEKERNEL->conveyor->wait_for_idle();
 
-            int n = gcode->has_letter('I') ? gcode->get_value('I') : 0;
+            int n = gcode->has_letter('I') ? gcode->get_value('I') : 7;
```

Comment:
Get this from original version:
            if(n == 0) n = 7;
            probe_grid(n, x, y, gcode->stream);

Will fix with configured values